core/python2Action/build.gradle

```diff
@@ -0,0 +1,17 @@
+ext.dockerImageName = 'python2action'
+apply from: '../../gradle/docker.gradle'
+distDocker.dependsOn 'copyFiles'
+distDocker.finalizedBy('rmFiles')
+
+task copyFiles(type: Copy) {
+    from('../actionProxy/actionproxy.py') {
+    }
+    from('../pythonAction/pythonrunner.py') {
+    }
+    into '.'
+}
+
+task rmFiles(type: Delete) {
+    delete 'actionproxy.py', 'pythonrunner.py'
+}
```

Comment:
As per an earlier comment, this should be solved more programmatically with information we already have:

```
def runners = files(
    new File(project(':core:actionProxy').projectDir, 'actionproxy.py'),
    new File(project(':core:pythonAction').projectDir, 'pythonrunner.py')
)
task copyFiles(type: Copy) {
    from runners
    into '.'
}

task rmFiles(type: Delete) {
    delete runners.collect { it.getName() }
}
```

The *finalizedBy* bit should leave out the parentheses for consistency